### DISCUSSION TITLE
Wrong version when using composer self-update on stable channel?

### DISCUSSION THREAD

**Discussion by jnoordsij** [composer/composer#10801] [Q&A / Support] [answered]
I have an update script on my local development machines that includes a run of `composer self-update`  to make sure I always use the latest Composer version locally. However, this morning I noticed the script updated my Composer version from 2.3.5 to 2.2.13:
```
+ sudo composer self-update
Upgrading to version 2.2.13 (stable channel).

Use composer self-update --rollback to return to version 2.3.5
```

Is this intended behavior, and is there some way to force the minor version not to be downgraded? Or is this somehow a bug in the updating code?

**Comment by Seldaek** [comment, marked as answer]
Thanks for reporting it. It's fixed now. See https://github.com/composer/getcomposer.org/commit/e5deaee38e51e2725c5f237635e2e3e907241e60
